Makefile

```diff
@@ -1,6 +1,23 @@
 UNIT_TAGS :=  "$(subst :, or ,$(shell awk '{print $2}' tests/unit.tags | paste -s -d: -))"
 INTEGRATION_TAGS := "$(subst :, or ,$(shell awk '{print $2}' tests/integration.tags | paste -s -d: -))"
 
+generate-init:
+	python -m scripts.generate_init
+
+check-generate-init:
+	python -m scripts.generate_init --check
+
+lint: check-generate-init
```

Comment:
I think it would be nice if these commands were specified in the README as well!

Reply:
In the development section, maybe we can just combine the `black` step and `mypy` step with the equivalent `make` command